docs/scan-image.md

```diff
@@ -0,0 +1,138 @@
+├─────────┼───────────────────┼───────────────┼────────────┼──────────────────┼───────────────┤
+│ stdlib  │ 1.21.0            │ Fix Available │         20 │ # 8 Layer        │ --            │
+╰─────────┴───────────────────┴───────────────┴────────────┴──────────────────┴───────────────╯
+╭─────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ Source:artifact:src/tester                                                                                      │
+├─────────────────────────────┬───────────────────┬───────────────┬────────────┬──────────────────┬───────────────┤
+│ PACKAGE                     │ INSTALLED VERSION │ FIX AVAILABLE │ VULN COUNT │ INTRODUCED LAYER │ IN BASE IMAGE │
+├─────────────────────────────┼───────────────────┼───────────────┼────────────┼──────────────────┼───────────────┤
+│ github.com/gogo/protobuf    │ 1.3.1             │ Fix Available │          1 │ # 9 Layer        │ --            │
+│ github.com/ipfs/go-bitfield │ 1.0.0             │ Fix Available │          1 │ # 9 Layer        │ --            │
+│ stdlib                      │ 1.19.8            │ Fix Available │         25 │ # 9 Layer        │ --            │
+╰─────────────────────────────┴───────────────────┴───────────────┴────────────┴──────────────────┴───────────────╯
+Debian:12
+╭───────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+│ Source:os:var/lib/dpkg/status                                                                             │
+├─────────────┬───────────────────┬─────────────────────────┬────────────┬──────────────────┬───────────────┤
+│ PACKAGE     │ INSTALLED VERSION │ FIX AVAILABLE           │ VULN COUNT │ INTRODUCED LAYER │ IN BASE IMAGE │
+├─────────────┼───────────────────┼─────────────────────────┼────────────┼──────────────────┼───────────────┤
+│ aom         │ 3.6.0-1+deb12u1   │ No fix available        │          2 │ # 1 Layer        │ --            │
+...
+│ zlib        │ 1:1.2.13.dfsg-1   │ No fix available        │          1 │ # 0 Layer        │ debian        │
+╰─────────────┴───────────────────┴─────────────────────────┴────────────┴──────────────────┴───────────────╯
+
+Filtered Vulnerabilities:
+╭─────────────┬───────────┬───────────────────────┬─────────────────────┬────────────────╮
+│ PACKAGE     │ ECOSYSTEM │ INSTALLED VERSION     │ FILTERED VULN COUNT │ FILTER REASONS │
+├─────────────┼───────────┼───────────────────────┼─────────────────────┼────────────────┤
+│ apt         │ Debian:12 │ 2.6.1                 │                   1 │ Unimportant    │
+│ binutils    │ Debian:12 │ 2.40-2                │                   8 │ Unimportant    │
+...
+│ util-linux  │ Debian:12 │ 2.38.1-5+deb12u2      │                   1 │ Unimportant    │
+╰─────────────┴───────────┴───────────────────────┴─────────────────────┴────────────────╯
+```
+
+</details>
+
+### Detailed Output:
```

Comment:
nit: this should go first, before the table output, since I think this is a pretty nice game changer.

Reply:
Because everything in the details is collapsed, it's actually just one line down. I'll make the screenshot no longer collapsed in <detail> to draw peoples eyes a bit more.

Reply:
Ah, not sure I fully understood your reponse, but I was suggesting we promote the HTML output first (since it's the nicest) in the documentation, then the CLI output. WDYT?

Reply:
Hmm... I think it's better to have the default output first, and we can still have that while promoting the HTML output since it's so close and takes up much more space on the docs page. See https://another-rex.github.io/osv-scanner/usage/scan-image#output, you can easily see the HTML output as the primary thing we want people to use, while we still have the first line explaining the default behaviour.

Reply:
SGTM. make sense now that I look at the final rendered output!